### DISCUSSION TITLE
Loading fonts from files

### DISCUSSION THREAD

**Discussion by HaanstootZA** [picoe/Eto#2114] [Ideas]
Hi

I am busy converting some winforms to Eto, but some of my code uses a font file.
Are there any plans to allow Eto to load from font files?

`
var fontCollection = new PrivateFontCollection()
var heighInPixels = 12;
fontCollection.AddFontFile(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "glyphfont.ttf"));
return new Font(fontCollection.Families[0], heightInPixels, GraphicsUnit.Pixel);`

**Comment by cwensley** [comment]
Hey @HaanstootZA,

Thanks for the idea!  This might be possible but is complicated to make work across all of the platforms in an easy way.  I have had other requests for this and will probably be getting around to it in the near future.

**Reply by HaanstootZA** [reply]
Hey @cwensley, thank you for getting back to me so quickly, this was my first time starting a discussion or requesting a feature on an open source project, and seeing that you not only responded but also accepted the suggestion left me feeling elated.

**Reply by rafntor** [reply]
+1 .. This project has a very friendly atmosphere for sure !

**Comment by cwensley** [comment]
See #2120 for details of the issue tracking this feature.

**Comment by cwensley** [comment]
FYI, this is now complete and should land in 2.6.2!  Thanks again for the feedback everyone.
